packages/inventory/src/models/reservation-item.ts

```diff
@@ -18,6 +24,9 @@ export class ReservationItem extends SoftDeletableEntity {
   @Column()
   quantity: number
 
+  @DbAwareColumn({ type: "enum", enum: ReservationType, default: ReservationType.INTERNAL })
+  type: ReservationType
+
```

Comment:
thought: would love if we could avoid this - looks like it is not currently being used?

I would be fine with an "external_id" prop if that would be helpful.